Apromore-Frontend/src/processmap/bpmnModelWrapper.js

```diff
@@ -45,7 +45,8 @@ export default class BPMNModelWrapper {
                 window.Apromore.I18N.Undo.undo,
                 window.Apromore.I18N.Undo.redo,
                 window.Apromore.I18N.Share.share,
-                window.Apromore.I18N.SimulationPanel.toggleSimulationDrawer
+                window.Apromore.I18N.SimulationPanel.toggleSimulationDrawer,
+                window.Apromore.I18N.SimulationPanel.simulateModel

```

Comment:
Can we remove simulationPanel.js and references to it as it is now replaced with simulateModel.js?